src/app.cpp

```diff
@@ -151,6 +154,7 @@ int main(int argc, char* argv[]) {
 
 	args::Positional<std::string> inarg(parser, "infile", "Input File", args::Options::Required);
 	args::Positional<std::string> outarg(parser, "outfile", "Output File", args::Options::Required);
+	args::Flag drop(parser, "drop", "Drop unknown packets", {'d', "drop"}, args::Options::Single);
```

Comment:
Rename to "tecmp-only"
```suggestion
	args::Flag tecmp_only(parser, "tecmp-only", "Only process TECMP packets, drop the others", args::Options::Single);
```